### ISSUE TITLE
SC2081 should check the != operator for globbing issue

### ISSUE THREAD

**Issue by lzutao** [koalaman/shellcheck#1374]
#### New checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/sh

if [ "$(tty)" != /dev/* ]; then
  echo 0
else
  echo 1
fi
```

#### Here's what shellcheck currently says:

No issues detected!

#### Here's what I wanted or expected to see:

Just like the warning SC2081:

```
[ .. ] can't match globs. Use [[ .. ]] or grep.
```

**Comment by koalaman** [comment]
A glaring omission. Well caught, thanks!
